docs/runtime-agent/03-02-configuring-runtime.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Configuring the Runtime
+type: Details
+---
+
+> **NOTE:** To represent API and Event Definitions of the connected Applications on Runtime, Open Service Broker API usage is recommended.
+
+In Kyma Runtime, during Runtime configuration, Application's Packages are integrated into the [Service Catalog](components/service-catalog) using [Application](components/application-connector#custom-resource-application) custom resources and [Application Broker](components/application-connector#architecture-application-broker).
+By default, a single Application is represented as a [ServiceClass](components/service-catalog/#architecture-resources), and a single Package is represented as a [ServicePlan](components/service-catalog/#architecture-resources) in the Service Catalog.
+Read more about [API Packages](https://github.com/kyma-incubator/compass/blob/master/docs/compass/03-packages-api.md) document.
+
+Runtime Agent periodically requests for the configuration of its Runtime from Compass.
+Changes in the configuration for the Runtime are applied by the Runtime Agent on the Runtime.
```

Comment:
```suggestion
Changes in the Runtime configuration are applied by the Runtime Agent on the Runtime.
```

Reply:
As discussed, I'm afraid this change does not retain the original meaning. I'll go with the original.